pynfft/cnfft3util.pxd

```diff
@@ -15,20 +15,19 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-from cnfft3 cimport fftw_complex
+from cnfft3 cimport fftw_complex, fftwf_complex, fftwl_complex
 
 cdef extern from "nfft3.h":
 
-    void nfft_vrand_unit_complex (fftw_complex *x, int n)
+    void nfft_vrand_unit_complex(fftw_complex *x, int n)
  	    # Inits a vector of random complex numbers in \
         # $[0,1]\times[0,1]{\rm i}$ .
 
-    void nfft_vrand_shifted_unit_double (double *x, int n)
+    void nfft_vrand_shifted_unit_double(double *x, int n)
         # Inits a vector of random double numbers in $[-1/2,1/2]$ .
 
-    void nfft_voronoi_weights_1d (double *w, double *x, int M)
```

Comment:
I don't it is necessary to remove `voronoi_weights_1d`. That function still exists in recent releases: https://github.com/NFFT/nfft/blob/cf336307f8794510717584cdbf616bdcde860959/kernel/util/voronoi.c#L23

Reply:
on second look I don't see that function in `nfft3.h`, but it is in `infft.h`. Those two headers are both included in `api.h`. Note sure if we can just use api.h instead of nfft3.h?

Reply:
That rings a bell. I think I removed it due to a compiler error I got since the function was nowhere declared in `nfft3.h`.
I'll see if `api.h` is an option, thx for the pointer.

Reply:
Actually, including `infft.h` as well through `api.h` opens up the possibility to use the name mangling macros, which could help avoiding duplication.

Reply:
I'll check in the NFFT documentation what is recommended. Perhaps `infft.h` is not intended for use outside of the library, and will cause us a lot of breakage between versions. OTOH, `api.h` sounds pretty public.